api4/team.go

```diff
@@ -224,8 +225,7 @@ func patchTeam(c *Context, w http.ResponseWriter, r *http.Request) {
 	c.App.SanitizeTeam(*c.App.Session(), patchedTeam)
 
 	auditRec.Success()
-	auditRec.AddMeta("team_name", patchedTeam.Name)
-	auditRec.AddMeta("team_display", patchedTeam.DisplayName)
+	auditRec.AddMeta("patched", patchedTeam)
 	c.LogAudit("")
```

Comment:
Not from this specific PR, but is this line needed?

Reply:
That is another remnant of the legacy audit system.  We need to leave it in for now.